util/git/client.go

```diff
@@ -422,11 +422,14 @@ func (m *nativeGitClient) Fetch(revision string) error {
 func (m *nativeGitClient) LsFiles(path string, enableNewGitFileGlobbing bool) ([]string, error) {
```

Comment:
```suggestion
func (m *nativeGitClient) LsFiles(pattern string, enableNewGitFileGlobbing bool) ([]string, error) {
```

Shouldn't this be `pattern` instead?

Reply:
it can be either. it depends on what is put inside the applicationset, but the globs are still representing a path.

Reply:
Both names are fine by me. I can change it to `pattern` if you think it's better, lmk

Reply:
If you look at the applicationset code, especially [this](https://github.com/argoproj/argo-cd/blob/master/applicationset/generators/git.go#L134), it is a little confusing whether it's pattern or path.